src/components/campaignguide/ScenarioStepComponent/ResolutionStepComponent.tsx

```diff
@@ -31,8 +33,11 @@ export default function ResolutionStepComponent({ step }: Props) {
       ) }
       { (!!resolution.text || resolution.steps.length > 0) && (
         <View style={space.marginTopM}>
-          <View style={space.marginSideM}>
-            <Text style={typography.mediumGameFont}>
+          <View style={{...space.marginSideM, display: 'flex', flexDirection: 'row', alignItems: 'center'}}>
+            { resolution.narration && (
```

Comment:
I think it might make sense to gate this control with a 'hasDissonantVoices' boolean.

Reply:
How should the `hasDissonantVoices` bool work? Done in redux?

Reply:
Yeah, I think redux state that just checks the `status` -- we can assume that the token is 'good' and not do the auth check until it is used, since being revoked is a very rare act.